bodo/libs/_query_profile_collector.cpp

```diff
@@ -193,9 +199,9 @@ static std::optional<boost::json::object> metric_to_json(MetricBase& metric) {
 }
 
 auto QueryProfileCollector::CollectSeenOperators()
-    -> std::unordered_map<operator_id_t, stage_id_t> {
+    -> std::map<operator_id_t, stage_id_t> {
     // Map all seen operators to the max stage value seen
-    std::unordered_map<operator_id_t, stage_id_t> seen_operator_stages;
+    std::map<operator_id_t, stage_id_t> seen_operator_stages;
```

Comment:
We allocate operator_ids in order and so most useful to print them in the same order in profile output.

Reply:
Let's add a comment in the code.

Reply:
Done.